ona/authentication.py

```diff
@@ -39,7 +40,8 @@ def get_api_token(json_web_token):
     # globally mainly because there isn't a test for it
     try:
         jwt_payload = decode_payload(json_web_token)
-        api_token = get_object_or_404(Token, key=jwt_payload.get('api-token'))
+        api_token = Token.objects.using("kobocat").filter(
+            key=jwt_payload.get('api-token')).select_related('user')
```

Comment:
I'd suggest we use `.get` rather than `.filter` so that:
- this fails with a 404 if token doesn't exist
- we don't have to use indexes - `api_token[0]` - to retrieve an object form the querylist